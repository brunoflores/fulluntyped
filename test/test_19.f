-- A record that fails to type-check due to missing field.

Counter = { get: Unit -> Nat, inc: Unit -> Unit };
c = let r = { x = ref 1 } in
      { get = lambda _: Unit. !(r.x) } as Counter;
