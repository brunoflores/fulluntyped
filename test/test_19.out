Counter: { get: Unit->Nat, inc: Unit->Unit }

4 | c = let r = { x = ref 1 } in
5 |       { get = lambda _: Unit. !(r.x) } as Counter;
                                            ^

field inc not in record provided here


